external-crates/move/crates/move-compiler/tests/move_2024/parser/labeled_lambda_body_invalid.exp

```diff
@@ -20,15 +50,23 @@ error[E01002]: unexpected token
    ┌─ tests/move_2024/parser/labeled_lambda_body_invalid.move:15:46
    │
 15 │         call!(|x| -> u64 'a: { return 'a x } + 1); // parsing error, lambdas cant appear in binop
-   │              -                               ^ Expected ')'
-   │              │                                
-   │              To match this '('
+   │                                              ^
+   │                                              │
+   │                                              Unexpected '+'
+   │                                              Expected ','
```

Comment:
A nit on these: we can pretty precisely tell the user which things we expected: anything in the start set plus the end token. This gets too large sometimes to bother reporting (the `exp` start set, for example), but here we could say "Expected `,` or `)`". I added a macro `format_oxford_list` under `shared/mod.rs` that I wrote originally precisely for these sorts of errors.

Reply:
Done!